src/utils/misc.ts

```
@@ -4,6 +4,11 @@ import { getMainMessage } from "./embeds"
 
 import type { Guild, GuildPremiumTier, VoiceChannel } from "discord.js"
 
+const compareArrays = (arr1: string[], arr2: string[]) => {
+    return arr1.length === arr2.length &&
+        arr1.every((val, index) => val === arr2[index]);
+};
```

ESLint

[error] Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`.

(arrow-body-style)